source/patterns/@aws-solutions-constructs/aws-s3-stepfunctions/lib/index.ts

```diff
@@ -107,56 +103,40 @@ export class S3ToStepfunctions extends Construct {
 
     let bucket: s3.IBucket;
 
+    if (props.deployCloudTrail !== undefined) {
+      defaults.printWarning("The deployCloudTrail prop has been deprecated since it no longer requires CloudTrail.");
```

Comment:
Think more when you're writing comments and docs. "it" in this sentence refers to the construct, but we never mention the construct so it makes no sense - and appears to refer to the deployCloudTrailProp.

"The deployClouTrail prop has been deprecated since this construct no longer requires AWS CloudTrail to implement its functionality. This construct no longer creates a CloudTrail in your account."